qml/components-Sailfish/Theme.qml

```diff
@@ -9,11 +9,11 @@ QtObject {
         switch (type) {
         case 3:
         case 4:
-            return "orange";
+            return "#fd9827";
         case 5:
-            return "green";
+            return highlightColor;
```

Comment:
Oh no, on Sailfish this is the current Theme color. I would prefer to use the green value from the website (or
put in the the BVApp color code)